[Change] Update information governance from WP3 doc (#164)
sa-tre/satre-specification: docs/source/pillars/information_governance.md

# Information governance

```{figure} ../../images/Capability_Map/full.drawio.svg
:alt: SATRE Pillars Capability Map
:align: center

SATRE Pillars Capability Map
```

This pillar concerns actions taken by the TRE operator to ensure information risk is measured and managed to an acceptable level.

Each TRE operator will have its own information governance requirements.
These will be informed by the context of the organisation, the work it performs and the nature of the data it processes.
For example, some requirements will arise from national legislation such as GDPR, discipline specific regulation like GCP, or contractural requirements from a specific data provider such as a company or research partner organisation.

## Governance Requirements

> **Requirements Gathering and Monitoring:** The process of collecting, documenting, and managing the functional and non-functional requirements for the TRE based on the TRE organisation's goals and data assets.

Author: New

```{list-table}
:header-rows: 1
:name: tab-ig-requirements-gathering-monitoring

* - Statement
  - Guidance
  - Importance
* - You must gather and monitor the information governance requirements needed to fulfil the legal and ethical standards to protect data subjects and deliver valuable research.
  - Requirements will come from a variety of sources including legislation, contractual obligations and ethical standards.
    Requirements must be monitored to ensure the TRE controls continue to match such requirements.
    New types of analysis or sources of data being brought into the scope of the TRE should be monitored to ensure requirements are current.
  - Mandatory
```

> **Controls:** Measures, safeguards, or mechanisms implemented to manage or mitigate risks and ensure the integrity, confidentiality, availability, and reliability of systems, processes, or data.

Author: New

```{list-table}
:header-rows: 1
:name: tab-ig-controls

* - Statement
  - Guidance
  - Importance
* - You must ensure controls are implemented to ensure the requirements are met.
  - Control implementation should be systematic and directly aligned to the internal and stakeholder requirements.
  - Mandatory
```

> **Resource Allocation:** The process of assigning, distributing, and managing resources (such as personnel, finances, equipment, or time) within the TRE organisation to meet objectives and priorities effectively.

Author: New

```{list-table}
:header-rows: 1
:name: tab-ig-resource-allocation

* - Statement
  - Guidance
  - Importance
* - You must ensure there are adequate resources to provide assurance and meet information governance requirements.
  - There should be adequate access to funding to implement and maintain the TRE which is under the direct control of the TRE organisation.
  - Mandatory
```

## Quality Management

What the organisation does to measure and control quality of processes, documentation and outputs.

> **Document and SOP Management:** Creating, organising, updating, and controlling documents and Standard Operating Procedures (SOPs) within the TRE organisation.

Author: Modified version of https://satre-specification.readthedocs.io/en/v0.2.0/pillars/information_governance.html#document-management

```{list-table}
:header-rows: 1
:name: tab-ig-quality-management

* - Statement
  - Guidance
  - Importance

> * - You must ensure all policies & standard operating procedures relevant to the TRE organisation are controlled.

Reviewer: ```suggestion
* - You must ensure all policies and standard operating procedures relevant to the TRE organisation are controlled.
```

>   - This may include measures like restricting edit access to relevant documents and recording acceptance of policies for all TRE organisation members.

Reviewer: Who are TRE organisation members?

>   - This may include measures like restricting edit access to relevant documents and recording acceptance of policies for all TRE organisation members.

Reviewer: Who are TRE organisation members?

  - Mandatory

> * - You must ensure all policies & standard operating procedures relevant to the TRE organisation are version controlled and have codified change processes.

Reviewer: ```suggestion
* - You must ensure all policies and standard operating procedures relevant to the TRE organisation are version controlled and have codified change processes.
```

  - Version control includes recording dates of changes, person responsible for carrying out changes, and summary of changes.
  - Mandatory
```

> **Quality Management Reporting:** The generation and dissemination of reports or dashboards that provide insights and metrics on the performance and effectiveness of quality management processes and activities.

Author: New

```{list-table}
:header-rows: 1
:name: tab-ig-quality-management-reporting

* - Statement
  - Guidance
  - Importance

> * - You must measure the performance of information governance within the TRE with regular reporting available to top management.

Reviewer: Who are top management?

Author: Maybe "senior management"? What's the generic term for people who have ultimate responsibility?

Reviewer: I think @machintim meant something like "the people with overall responsibility for the TRE organisation" where "TRE organisation" is defined in #160. Worth adding to the glossary and/or finding a better name?

Reviewer: ```suggestion
* - You must measure the performance of information governance within the TRE with regular reporting available to the TRE organisation management team.
```

  - This may include reports and dashboards showing security incidents, quality management deviations and audit findings.
  - Mandatory
```

> **Internal Audit:** An independent evaluation process within the TRE organisation that assesses and improves its internal controls, risk management, and governance.

Author: Based on https://satre-specification.readthedocs.io/en/v0.2.0/pillars/information_governance.html#compliance-monitoring-and-reporting

```{list-table}
:header-rows: 1
:name: tab-ig-internal-audit

* - Statement
  - Guidance
  - Importance
* - You must audit your TRE organisation against relevant requirements and standards
  - If you are publicly accredited against a standard, for instance ISO27001, DSPT, CE+ etc., you must have processes in place to ensure you remain compliant.
  - Mandatory
* - You must report on and share outcomes of each audit of your TRE organisation with the required bodies.
  - This may include regulatory bodies or the organisations that manage accreditations you have
  - Mandatory
```

> **Supplier Management and Monitoring:** A structured approach to managing and monitoring relationships with external suppliers, vendors and contractors, including selection, contract management and compliance oversight.

Author: New

```{list-table}
:header-rows: 1
:name: tab-ig-supplier-management-monitoring

* - Statement
  - Guidance
  - Importance

> * - You must ensure that suppliers, contractors and sub-contractors align with the security requirements of you TRE.

Reviewer: Suppliers of what? Does this apply to eg. cleaners?

Author: Maybe IT suppliers? @machintim what do you think?

Reviewer: ```suggestion
* - You must ensure that suppliers, contractors and sub-contractors with access to your TRE align with your security requirements.
```

  - These should be included as mandatory, non-functional requirements in during procurement and contracting.
    This will also include contractor staff contracts for example, legal liability and NDAs.
  - Mandatory
* - You must monitor compliance of your suppliers with the terms of the contracts.
  - This will include monitoring changes in the services and infrastructure being delivered and quality management within the contractor’s organisation.
    This may be done through formal audit or by monitoring change and quality documentation provided by the supplier.
  - Mandatory
```

> **Asset Management Process:** A systematic approach to acquiring, operating, maintaining, and disposing of assets within an organization, aimed at maximizing their value and minimizing risks.

Author: New

```{list-table}
:header-rows: 1
:name: tab-ig-asset-management-process

* - Statement
  - Guidance
  - Importance

> * - You must track and maintain physical assets if used to ensure security is maintained.
>   - Safe disposal of storage media and warranties maintained for critical infrastructure-maintained confidentiality, integrity and availability.
>   - Mandatory (where physical assets are in scope)

Reviewer: ```suggestion
* - You must track and maintain any physical assets used by your TRE.
  - All physical assets should be maintained and covered by warranty if applicable
    At the end of their lifetime, assets should be securely disposed of in such a way that data cannot be recovered from them.   
  - Mandatory (where physical assets are in scope)
```

```

> **Issue Management Process:** A systematic approach to identifying, tracking, resolving, and managing issues or problems that arise within a TRE organisation, aiming to minimize their impact and ensure timely resolution.

Author: Modified from https://satre-specification.readthedocs.io/en/v0.2.0/pillars/information_governance.html#issue-management

```{list-table}
:header-rows: 1
:name: tab-ig-issue-management-process

* - Statement
  - Guidance
  - Importance

> * - You must log, track and resolves issues resulting from deviations from processes, incidents and audit findings.

Reviewer: ```suggestion
* - You must log, track and resolve any issues resulting from deviations from processes, incidents and audit findings.
```

>   - This process should be tracked through an electronic record and workflow system with records retained.

Reviewer: Why is it mandatory to do this electronically? A comprehensive paper system would be more useful than a cursory "stick everything in a folder on a shared drive" electronic one.

Author: I guess there's some ambiguity in how we're writing the spec. The `Statement` is mandatory, but arguably `Guidance` is always advisory?

Reviewer: ```suggestion
  - This process could, for example, be tracked through an electronic record and workflow system with records retained.
```

  - Mandatory
* - You must use reported issues to inform changes, such as for process improvement and risk management.
  - All issues should be analysed for their route cause and improvements put in place to prevent further occurrence.
  - Mandatory
```

> **Quality Management Data:** Data, including training records and configuration data, collected and used to monitor, evaluate, and improve the quality of processes, or services within the TRE organisation.

Author: New

```{list-table}
:header-rows: 1
:name: tab-ig-quality-management-data

* - Statement
  - Guidance
  - Importance
* - You must collect and maintain quality management data for measuring the effectiveness of a TRE.
  - Large amounts of data will be produced by elements within the TRE.
    These data should be analysed with reports and dashboards provided to guide TRE implementer’s improvements and provide re-assurance to data consumers and subjects.

>   - Mandatory

Reviewer: Why is this mandatory?

```

> **Electronic Quality Management System Application:** A software application or platform used to manage and automate quality management processes, including document control, corrective actions, audits, and performance tracking.

Author: New

```{list-table}
:header-rows: 1
:name: tab-ig-electronic-quality-management-system-application

* - Statement
  - Guidance
  - Importance
* - You could use an eQMS (Electronic Quality Management System) to standardise and automate quality management tasks and workflows, and to generate quality data and reports automatically.
  - A basis eQMS could be a set of spreadsheets or documents held in a repository which are manually maintained.
    More mature applications will provide workflows and generate quality data through manual and automated actions.
  - Optional
```

## Risk Management

What the organisation does to ensure information risk is measured and managed to an acceptable level.

> **Risk Assessment:** The systematic evaluation and analysis of potential risks, threats, or vulnerabilities, including their likelihood, potential impact, and the effectiveness of existing controls or mitigation measures.

Author: Modified from https://satre-specification.readthedocs.io/en/v0.2.0/pillars/information_governance.html#risk-management , split into separate assessment and treatment sections

```{list-table}
:header-rows: 1
:name: tab-ig-risk-assessment

* - Statement
  - Guidance
  - Importance
* - You must have a way to score risk to understand the underlying severity.
  - You have a risk assessment methodology for scoring risks on multiple axes such as impact and likelihood.
  - Mandatory

> * - You must carry out a data processing assessment for all projects requiring a TRE that are working with sensitive data.

Reviewer: Do we need to define sensitive data?

  - A data processing assessment is a process designed to identify risks arising out of the processing of sensitive data and to minimise these risks as far and as early as possible. This may take the form of an existing regulatory requirements such as Data Protection Impact Assessment.
  - Mandatory
```

> **Risk Treatment:** The selection and implementation of strategies, controls, or measures to manage or mitigate identified risks, such as risk avoidance, risk transfer, risk reduction, or risk acceptance.

Author: Modified from https://satre-specification.readthedocs.io/en/v0.2.0/pillars/information_governance.html#risk-management , split into separate assessment and treatment sections

```{list-table}
:header-rows: 1
:name: tab-ig-risk-treatment

* - Statement
  - Guidance
  - Importance
* - You must have a process for designing, implementing and recording risk mitigations where indicated by a risk assessment.
  - Actions that are taken or not taken following a risk assessment must be recorded.
  - Mandatory
```

> **Risk Ownership:** The assignment of responsibility and accountability to individuals or entities for managing and mitigating specific risks within the TRE organisation.

Author: Partly new, partly modified from https://satre-specification.readthedocs.io/en/v0.2.0/pillars/information_governance.html#risk-management

```{list-table}
:header-rows: 1
:name: tab-ig-risk-ownership

* - Statement
  - Guidance
  - Importance
* - You must have a clear set of roles and responsibilities relating to risk including who owns risks and how they are escalated and delegated.

>   - The highest level of risk ownership is the top management of the TRE organisation. In order to ensure escalations to this level are rare suitable structures should be put in place to own, mitigate and accept risk.

Reviewer: ```suggestion
  - The highest level of risk ownership is the top management of the TRE organisation.
    In order to ensure escalations to this level are rare, suitable structures should be put in place to own, mitigate and accept risk.
```

  - Mandatory

> * - You must understand risk appetite.

Reviewer: ```suggestion
* - You must understand your organisational risk appetite.
```

  - This includes understanding ownership of risk, and ability to accept risk which falls outside of the appetite should that become necessary.
  - Mandatory
```

> ## Study Management

Author: `Study Management` is a replacement for the `Project Management` section https://satre-specification.readthedocs.io/en/v0.2.0/pillars/information_governance.html#project-management

Onboarding and Closure is copied, `Compliance checking` is new.

  - Mandatory
```

**Resource Allocation:** The process of assigning, distributing, and managing resources (such as personnel, finances, equipment, or time) within the TRE organisation to meet objectives and priorities effectively.

```{list-table}
:header-rows: 1
:name: tab-ig-resource-allocation

* - Statement
  - Guidance
  - Importance
* - You must ensure there are adequate resources to provide assurance and meet information governance requirements.
  - There should be adequate access to funding to implement and maintain the TRE which is under the direct control of the TRE organisation.
  - Mandatory
```

## Quality Management

What the organisation does to measure and control quality of processes, documentation and outputs.

**Document and SOP Management:** Creating, organising, updating, and controlling documents and Standard Operating Procedures (SOPs) within the TRE organisation.

```{list-table}
:header-rows: 1
:name: tab-ig-quality-management

* - Statement
  - Guidance
  - Importance
* - You must ensure all policies and standard operating procedures relevant to the TRE organisation are controlled.

>   - This may include measures like restricting edit access to relevant documents and recording acceptance of policies for all TRE organisation members.

Reviewer: Can we define TRE organisation members?

Author: Can you think of a generic phrase for everyone running/operating/governing/using the TRE?

Reviewer: #160 has a definition of TRE organisation. Should we merge that as a starting point?

Author: OK, shall we work on the assumption that https://github.com/sa-tre/satre-specification/pull/160 will be merged soon?

Reviewer: I'm happy to merge it now (without most of the definitions) but if anyone (@machintim?) has some definitions they want to add, let's add those to #160 today.

  - Mandatory (where physical assets are in scope)
```

**Issue Management Process:** A systematic approach to identifying, tracking, resolving, and managing issues or problems that arise within a TRE organisation, aiming to minimize their impact and ensure timely resolution.

```{list-table}
:header-rows: 1
:name: tab-ig-issue-management-process

* - Statement
  - Guidance
  - Importance
* - You must log, track and resolve any issues resulting from deviations from processes, incidents and audit findings.
  - This process should be tracked through an electronic record and workflow system with records retained.
  - Mandatory
* - You must use reported issues to inform changes, such as for process improvement and risk management.
  - All issues should be analysed for their route cause and improvements put in place to prevent further occurrence.
  - Mandatory
```

**Quality Management Data:** Data, including training records and configuration data, collected and used to monitor, evaluate, and improve the quality of processes, or services within the TRE organisation.

```{list-table}
:header-rows: 1
:name: tab-ig-quality-management-data

* - Statement
  - Guidance
  - Importance

> * - You must collect and maintain quality management data for measuring the effectiveness of a TRE.
>   - Large amounts of data will be produced by elements within the TRE.
>     These data should be analysed with reports and dashboards provided to guide TRE implementer’s improvements and provide re-assurance to data consumers and subjects.
>   - Mandatory

Author: ```suggestion
* - You should collect and maintain quality management data for measuring the effectiveness of a TRE.
  - Large amounts of data will be produced by elements within the TRE.
    These data should be analysed with reports and dashboards provided to guide TRE implementer’s improvements and provide re-assurance to data consumers and subjects.
  - Recommended
```

:name: tab-ig-electronic-quality-management-system-application

* - Statement
  - Guidance
  - Importance
* - You could use an eQMS (Electronic Quality Management System) to standardise and automate quality management tasks and workflows, and to generate quality data and reports automatically.
  - A basis eQMS could be a set of spreadsheets or documents held in a repository which are manually maintained.
    More mature applications will provide workflows and generate quality data through manual and automated actions.
  - Optional
```

## Risk Management

What the organisation does to ensure information risk is measured and managed to an acceptable level.

**Risk Assessment:** The systematic evaluation and analysis of potential risks, threats, or vulnerabilities, including their likelihood, potential impact, and the effectiveness of existing controls or mitigation measures.

```{list-table}
:header-rows: 1
:name: tab-ig-risk-assessment

* - Statement
  - Guidance
  - Importance
* - You must have a way to score risk to understand the underlying severity.
  - You have a risk assessment methodology for scoring risks on multiple axes such as impact and likelihood.
  - Mandatory

> * - You must carry out a data processing assessment for all projects requiring a TRE that are working with sensitive data.

Author: Probably not, since you probably wouldn.t be using a TRE otherwise!
```suggestion
* - You must carry out a data processing assessment for all projects requiring a TRE.
```

<!-- See all pillars of the SATRE Pillars Capability Map here: {ref}`satre_pillars` -->

## Requirements

Information governance requirements are defined by top management within the TRE implementing organisation.
These will be draw from the context of the organisation, the work it performs and the nature of the data it processes.
Gathering and monitoring these requirements are a process for the capability along with the overall ownership of risk, definition of controls for those risks and the allocation of resources across the TRE organisation.
Quality data is a key input to measuring and improving the effectiveness of the TRE in relation to governance, risk and compliance.

These requirements will be different for every TRE organisation.

> TRE organisations should conduct a requirements gathering exercise to identity the applicability of each item in the specification, and to identity the best way to implement them.

Author: Maybe this paragraph should be in a pre-amble since it applies to all requirements for all pillars?

  - Mandatory
```

> **Study Management Portal:** An online platform that provides centralised access to manage research studies including onboarding studies, control of access and administration of compliance tasks.

Author: New

```{list-table}
:header-rows: 1
:name: tab-ig-study-management-portal

* - Statement
  - Guidance
  - Importance
* - You could implement a portal that can provide a workflow engine and database which automates the processes within this capability.
  - A portal should automate as much of the processes within the capability as possible.
    Where processes are automated process maturity is easier to achieve with processes being completed more consistently and producing quality control and monitoring data automatically
  - Optional
```

> **Data Asset Register:** A database or other electronic record that documents and manages information about the TRE organisation's data assets, including their characteristics, ownership, usage, and other relevant details.

Author: New

```{list-table}
:header-rows: 1
:name: tab-ig-data-asset-register

* - Statement
  - Guidance
  - Importance
* - You must keep a complete record of all the data assets held within the system.

>   - All data asset (current and past) held by the system should be retained along with meta-data useful for ensuring compliance can be demonstrated.

Reviewer: ```suggestion
  - Details of all data assets (current and past) held by the system should be retained along with meta-data useful for ensuring compliance can be demonstrated.
```

    This would include ownership, data lifecycle, contracts, risk assessments and other quality data.
    This is likely to already exist within the wider organisation but may require augmenting for the TRE.
  - Mandatory
```

> **Study Register:** A centralised record or database that tracks and manages information about research studies and projects.

Author: New

```{list-table}
:header-rows: 1
:name: tab-ig-study-register

* - Statement
  - Guidance
  - Importance
* - You must keep a complete record of all the research studies and projects within the TRE current and past.
  - The study register should contain all data related to a study including a reference to data assets, members (researchers, owners etc.) and any compliance activities required.

>   - Mandatory

Reviewer: Making this mandatory implies that if you ever fail to do this for a single project, you can never be a TRE.

Author: Are you objecting to the statement being mandatory, or the guidance?

Reviewer: Not sure why this is associated with the `onboarding processes` statement but it's meant to be the `study-register`. I'm objecting to making it mandatory to keep a complete record of all past, current and future projects. Records might be missing by accident (eg. legacy projects from before SATRE was adopted), or deliberately (eg. for confidentiality reasons) but I don't think this should cause you to fail your SATRE assessement.

Happy with this text at "Recommended" though.

  - This may include multi-factor authentication (MFA), ID checks or email/phone verification.
  - Mandatory
```

**User Onboarding:** The process of introducing and integrating researchers and data consumers onto a TRE's systems, processes, including training, access provisioning, and orientation.

```{list-table}
:header-rows: 1
:name: tab-ig-user-onboarding

* - Statement
  - Guidance
  - Importance
* - You must have clear onboarding processes in place for all roles within your TRE organisation.
  - This may include all members signing role-specific terms of use and completing role specific training.
  - Mandatory
```

**Identity and Access Management Services:** Govern and control user identities, access privileges, authentication, and authorization within an organisation.

```{list-table}
:header-rows: 1
:name: tab-ig-identity-access-management-services

* - Statement
  - Guidance
  - Importance
* - You must have a set of services to manage access to resources based on identity.
  - This will include a security model for role based access with technical controls to ensure the principle of least privilege is enforced.
  - Mandatory

> * - The data asset owner must have control of who has access to which assets.

Reviewer: What does "ownership" mean? Are we talking about the Data Owner? Typically TRE operators might be Controllers or Processors but are not necessarily owners of the data itself.

Author: Good point, I think it depends on what _control_ means. Does it mean the data owner has direct technical control of access via TRE permissions, or is it higher level governance in the sense that the data owner _controls_ access by telling the data controllers/processors who should be allowed access or what criteria should be used to decide access, and the controller/processor must implement that by the appropriate technical means?

Reviewer: Thoughts @machintim ?

Reviewer: ```suggestion
* - You must not give anyone access to datasets without agreement from the Data Controller.
```

  - Either through policy and process or ideally direct control access to data should be controlled by the person who owns a data asset or their delegates.
  - Mandatory
```

**Authentication Application:** A software system that verifies and validates the identities of users or entities accessing a system through multifactor authentication.

```{list-table}
:header-rows: 1
:name: tab-ig-authentication-application

* - Statement
  - Guidance
  - Importance
* - You must have robust and secure applications in place to authenticate users (and services) within the TRE.
  - The number of authentication applications should be kept to a minimum with common controls and standards applied across all such as MFA, password complexity etc.
  - Mandatory
```

**User Identity Attributes:** Characteristics or attributes associated with a user's identity, such as username, email address, role, permissions, or affiliations.

```{list-table}
:header-rows: 1
:name: tab-ig-user-identity-attributes

* - Statement
  - Guidance
  - Importance

> * - You must give each user of the TRE a unique logon with changes to any records strictly controlled.
>   - The unique identifier and all associated records for a user should be traceable across the entire TRE. This will include training records, affiliations, contract agreements and ethics approvals where required.
>   - Mandatory

Reviewer: ```suggestion
* - You must give each user of the TRE a unique logon with changes to any records strictly controlled.
  - The unique identifier and all associated records for a user should be traceable across the entire TRE. 
    This will include training records, affiliations, contract agreements and ethics approvals where required.
  - Mandatory
```

```

## Training Delivery and Management

Ability to deliver, track and maintain adequate training levels to ensure competence of all people within the TRE organisation.

**Curriculum Creation and Management:** The process of designing, developing, and managing educational curricula, courses through training needs analysis for required competency.

```{list-table}
:header-rows: 1
:name: tab-ig-curriculum-creation-management

* - Statement
  - Guidance
  - Importance

> * - You must have relevant training for all roles within the TRE organisation, and the ability to deliver this training.

Reviewer: Why should "you" (the TRE operator) have to deliver the training? Why not outsource or use a common accreditation (eg. ONS Safe Researcher).

Author: I assume "you" means "you're responsible for", but are free to delegate it.

Reviewer: ```suggestion
* - You must ensure that relevant training is available for all roles within the TRE organisation.
```

  - This may include, for instance, cyber security training, GDPR training, and higher level training for system operators.

>     Specific training should be designed for roles such as Senior Information Risk Owner, Data Asset owner. Identification of these specialities should be done through a systematic training needs analysis.

Reviewer: ```suggestion
    Specific training should be designed for roles such as Senior Information Risk Owner, Data Asset owner.
    Identification of these specialities should be done through a systematic training needs analysis.
```

    Specific training may also be required based on the data or data provider such as GCP.
  - Mandatory

> * - You should have a training platform capable of delivering online training in a variety of formats.

Reviewer: Should this be optional instead of recommended?

  - This should include competency testing and more simple recording of actions such as read and understood.
  - Recommended

> * - You must provided repeat or updated where necessary to account for changes in competency requirements.

Reviewer: ```suggestion
* - You must provide repeat or updated training where necessary to account for changes in competency requirements.
```

  - Training is not a one-off event.
    Electronic reminders for refresher training should be considered.
    Ideally training should remain relevant and so policies and processes should enable people to demonstrate competency rather than repeat training unnecessarily.
  - Mandatory
```

**Certification Management:** The process of managing and overseeing certifications or qualifications held by individuals or entities, including tracking expiry dates, renewals, and compliance requirements.

```{list-table}
:header-rows: 1
:name: tab-ig-certification-management

* - Statement
  - Guidance
  - Importance
* - You must maintain accurate training records that are directly tied to the role and access levels within the TRE.

>   - Electronic training records should be tied to a user record and carefully maintained.

Reviewer: Why require electronic?

Author: Do you know anyone who isn't maintaining electronic records? Would you trust them with your electronic data :smile: ?

I've removed it anyway.
```suggestion
  - Training records should be tied to a user record and carefully maintained.
```

  - Mandatory

> * - You should accept proof of relevant training certifications from third parties.

Reviewer: This directly contradicts the earlier requirement to run training yourself.

Reviewer: Could be resolved if suggestion above is accepted.

Author: Accepted the above suggestion

  - Optional
```

**Data Asset Register:** A database or other electronic record that documents and manages information about the TRE organisation's data assets, including their characteristics, ownership, usage, and other relevant details.

```{list-table}
:header-rows: 1
:name: tab-ig-data-asset-register

* - Statement
  - Guidance
  - Importance
* - You must keep a complete record of all the data assets held within the system.
  - Details of all data assets (current and past) held by the system should be retained along with meta-data useful for ensuring compliance can be demonstrated.
    This would include ownership, data lifecycle, contracts, risk assessments and other quality data.
    This is likely to already exist within the wider organisation but may require augmenting for the TRE.
  - Mandatory
```

**Study Register:** A centralised record or database that tracks and manages information about research studies and projects.

```{list-table}
:header-rows: 1
:name: tab-ig-study-register

* - Statement
  - Guidance
  - Importance

> * - You must keep a complete record of all the research studies and projects within the TRE current and past.
>   - The study register should contain all data related to a study including a reference to data assets, members (researchers, owners etc.) and any compliance activities required.
>   - Mandatory

Author: ```suggestion
* - You should keep a complete record of all the research studies and projects within the TRE current and past.
  - The study register should contain all data related to a study including a reference to data assets, members (researchers, owners etc.) and any compliance activities required.
  - Recommended
```

  - Mandatory
```

**User Onboarding:** The process of introducing and integrating researchers and data consumers onto a TRE's systems, processes, including training, access provisioning, and orientation.

```{list-table}
:header-rows: 1
:name: tab-ig-user-onboarding

* - Statement
  - Guidance
  - Importance
* - You must have clear onboarding processes in place for all roles within your TRE organisation.
  - This may include all members signing role-specific terms of use and completing role specific training.
  - Mandatory
```

**Identity and Access Management Services:** Govern and control user identities, access privileges, authentication, and authorization within an organisation.

```{list-table}
:header-rows: 1
:name: tab-ig-identity-access-management-services

* - Statement
  - Guidance
  - Importance
* - You must have a set of services to manage access to resources based on identity.
  - This will include a security model for role based access with technical controls to ensure the principle of least privilege is enforced.
  - Mandatory
* - The data asset owner must have control of who has access to which assets.

>   - Either through policy and process or ideally direct control access to data should be controlled by the person who owns a data asset or their delegates.

Reviewer: ```suggestion
  - The Data Controller may choose to delegate this authority.
```

```{list-table}
:header-rows: 1
:name: tab-ig-user-identity-attributes

* - Statement
  - Guidance
  - Importance
* - You must give each user of the TRE a unique logon with changes to any records strictly controlled.
  - The unique identifier and all associated records for a user should be traceable across the entire TRE.
    This will include training records, affiliations, contract agreements and ethics approvals where required.
  - Mandatory
```

## Training Delivery and Management

Ability to deliver, track and maintain adequate training levels to ensure competence of all people within the TRE organisation.

**Curriculum Creation and Management:** The process of designing, developing, and managing educational curricula, courses through training needs analysis for required competency.

```{list-table}
:header-rows: 1
:name: tab-ig-curriculum-creation-management

* - Statement
  - Guidance
  - Importance
* - You must have relevant training for all roles within the TRE organisation, and the ability to deliver this training.
  - This may include, for instance, cyber security training, GDPR training, and higher level training for system operators.

>     Specific training should be designed for roles such as Senior Information Risk Owner, Data Asset owner.
>     Identification of these specialities should be done through a systematic training needs analysis.
>     Specific training may also be required based on the data or data provider such as GCP.

Reviewer: Since we haven't written down these roles anywhere I suggest the following:

```suggestion
    Specialised roles are likely to need more tailored training.
    Identification of these specialities should be done through a systematic training needs analysis.
    Specific training may also be required based on the data or data provider such as GCP.
```

  - Mandatory

> * - You should have a training platform capable of delivering online training in a variety of formats.
>   - This should include competency testing and more simple recording of actions such as read and understood.
>   - Recommended

Author: ```suggestion
* - You could have a training platform capable of delivering online training in a variety of formats.
  - This could include competency testing and more simple recording of actions such as read and understood.
  - Optional
```
